components/interactive_dialog/dialog_element/dialog_element.tsx

```diff
@@ -5,18 +5,20 @@ import React from 'react';
 
 import {FormattedMessage} from 'react-intl';
 
-import MenuActionProvider from 'components/suggestion/menu_action_provider';
-import GenericUserProvider from 'components/suggestion/generic_user_provider.jsx';
+import GenericUserProvider from 'components/suggestion/generic_user_provider';
+
```

Comment:
nit pick: no extra lines between same group of import. In this case, these are all from `components` so no need to put extra lines